internal/plan/plan.go

```diff
@@ -276,6 +294,11 @@ func ParseLayer(order int, label string, data []byte) (*Layer, error) {
 				Message: fmt.Sprintf("cannot use reserved service name %q", name),
 			}
 		}
+		if service == nil {
+			return nil, &FormatError{
+				Message: fmt.Sprintf("service object cannot be null for service %q", name),
```

Comment:
This would also be true if 'service == ""' wouldn't it?
Would it be better to do something like:

`if service == nil || service == "" `
?

Reply:
`service` is a `Service` struct, so comparing it to `""` doesn't make sense. But I've added a check for the service name being empty string above as well.